dev/sbt-plugin/src/sbt-test/sbt-plugin/akka-remote-in-dev/build.sbt

```diff
@@ -37,13 +31,3 @@ lazy val `b-impl` = (project in file("b") / "impl")
     libraryDependencies ++= Seq(lagomScaladslCluster, macwire, netty)
   )
   .dependsOn(`b-api`)
-
-
-InputKey[Unit]("assertRequest") := {
```

Comment:
`ScriptTools` already provide tasks to make requests and assertions about the response so that we can drop these custom tasks.

Reply:
Is this version of `assertRequest` identical to that provided on `ScriptTools`? I think there's overrides/customizations.

Reply:
The only version that cannot be replaced is the one for SSL tests. These others are similar enough to be discarded.

Reply:
The only version that cannot be replaced is the one for SSL tests. These others are similar enough to be discarded.